.storybook/Banner.js

```diff
@@ -0,0 +1,54 @@
+import React from 'react'
+import { storiesOf } from '@storybook/react'
+import { Box, Banner } from '../src'
+
+storiesOf('Banner', module)
+  .add('All palettes', () => (
+    <Box>
+      <Banner align='right' mb={2} p={3} title='DEFAULT' subtitle='Secondary Text' />
```

Comment:
I am pretty sure that we will have to change this in the future and have some design additions (adding a closing icon for example..). I would vote to use a [Compound Component](https://github.com/ReactTraining/react-workshop/blob/master/subjects/CompoundComponents/lecture.js) (now I know what does that mean :)). Maybe the Banner should extend a Card and create something like

```
<Banner palette='warning'>
  <BannerTitle></BannerTitle>
  <BannerSubtitle></BannerSubtitle>
</Banner>
```